scripts/sts/sts-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
I had done something similar with an expected secret but put it into the config/secrets folder and used templating for providing the ARN https://github.com/integr8ly/cloud-resource-operator/pull/215/files#diff-dddcdd3e58af2f907e4c8812ca7953563c29620af6205c38aea99dcc66b0eeb0R1

Reply:
Although seems like you are doing lots more in the script so this is probably better.

Reply:
The script is not using this secret actually.
I am happy to move it to a Template in config/secrets.